src/modelinstance.cpp

```diff
@@ -237,6 +237,34 @@ Status ModelInstance::loadOVCNNNetwork() {
     return StatusCode::OK;
 }
 
+Status ModelInstance::loadOVCNNNetworkUsingCustomLoader() {
+    spdlog::debug("Try reading model using a custom loader");
```

Comment:
please use macro SPDLOG_DEBUG in all places.

Reply:
In the same file in existing code spdlog::debug is used at multiple places. Wanted to be uniform in the file. If needed, we can change everywhere in the file.

Reply:
Ok, this is low priority, could be fixed later.